src/Internal/Wallet/Cip30.purs

```diff
@@ -40,18 +36,31 @@ import Untagged.Union (asOneOf)
 type Cip30Wallet =
   { -- A reference to a connection with the wallet, i.e. `window.cardano.nami`
     connection :: Cip30Connection
-  -- Get the address associated with the wallet (Nami does not support
-  -- multiple addresses)
-  , getWalletAddresses :: Cip30Connection -> Aff (Maybe (Array Address))
-
-  -- Get combination of all available UTxOs
-  , getBalance :: Cip30Connection -> Aff (Maybe Value)
+  -- Returns the network id of the currently connected account. 0 is testnet 
+  -- and 1 is mainnet but other networks can possibly be returned by wallets.
```

Comment:
> but other networks can possibly be returned by wallets

This is not true. CDDL specifies network_id as strictly 0 or 1: https://github.com/input-output-hk/cardano-ledger/blob/677bf5825c34f4f9be70ca45fc836b2b72136451/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L384

Reply:
thanks! 

I will change it to return `NetworkId` then and modify the comment to : 

`In principle CIP30 allows wallets to return more networks but CDDL for Alonzo only allows tesnet or mainnet` (of course including the links for both)

Reply:
It's not true either. CIP-30 does not allow to return "more networks", because there are no more network IDs possible.